app/serializers/discourse_activity_pub/ap/activity/announce_serializer.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class DiscourseActivityPub::AP::Activity::AnnounceSerializer < DiscourseActivityPub::AP::ActivitySerializer
+end
```

Comment:
Yeah. Activities are wrapped in Announce activities so that the Category Actor can share Activities it didn't perform itself. That happens here: https://github.com/discourse/discourse-activity-pub/pull/22/files#diff-aa0d4037d9999b3e343873e9b9a4327c964f41dd12d023754541b71e3f3962c5R95